### PR TITLE
 append https to the aqua/config/gateway-external value

### PR DESCRIPTION
**Pull request by anshupande**
Greetings and thanks for the PR.  We have a few rules so that everyone enjoys your Pull Request.

## Notes

You must explain how any `docker run` executions your PR introduces handles log accumulation. 

Did you introduce any command that executes `docker run`?

Are there any dependencies on github.com/adobe-platform/infrastructure?

If so:
 
Is an update to the base stack required(rare)?

Is an A/B required?

Are you dependent on new secrets, or infrastructure in any way?

### CODE REVIEW

**Comment by anshupande** [comment]
```
core@ip-10-74-131-124 sandbox-aquasec-us-east-1a-control ~ $ docker inspect flight-director |  grep AQUA
                "FD_AQUA_PROTOCOL=",
                "FD_AQUA_ENDPOINT=***",
                "FD_AQUA_USER=***",
                "FD_AQUA_PASSWORD=***",
``` 
There is no protocol being passed,so it defaults to `http` and the ELB expects `https`

File: v1/opt/aqua/setup/leader/002-defaults.sh [hunk 1]
```diff
@@ -11,4 +11,4 @@ etcd-set /aqua/config/aqua-token "aquatoken"
 etcd-set /aqua/config/user "administrator"
 etcd-set /aqua/config/password "password"
 etcd-set /aqua/config/gateway-host "$AQUA_INTERNAL_ELB"
-etcd-set /aqua/config/gateway-external "$AQUA_EXTERNAL_ELB"
+etcd-set /aqua/config/gateway-external "https://""$AQUA_EXTERNAL_ELB"
```
Comment (iderdik) [inline v1/opt/aqua/setup/leader/002-defaults.sh]: This looks odd, have you verified this behaves as expected. Would simply:
`etcd-set /aqua/config/gateway-external "https://$AQUA_EXTERNAL_ELB"` work too?

Comment (anshupande) [inline v1/opt/aqua/setup/leader/002-defaults.sh]: core@ip-10-74-131-124 sandbox-aquasec-us-east-1a-control ~ $ etcdctl set /aqua/config/gateway-external "https://""$AQUA_EXTERNAL_ELB"
```
https://aquasec-sa-AquaElb-PN36MQ2MP0NT-1376431810.us-east-1.elb.amazonaws.com
core@ip-10-74-131-124 sandbox-aquasec-us-east-1a-control ~ $ etcdctl get aqua/config/gateway-external
https://aquasec-sa-AquaElb-PN36MQ2MP0NT-1376431810.us-east-1.elb.amazonaws.com

```

Comment (anshupande) [inline v1/opt/aqua/setup/leader/002-defaults.sh]: this works as well: 

```
core@ip-10-74-131-124 sandbox-aquasec-us-east-1a-control ~ $ etcdctl set /aqua/config/gateway-external "https://$AQUA_EXTERNAL_ELB"
https://aquasec-sa-AquaElb-PN36MQ2MP0NT-1376431810.us-east-1.elb.amazonaws.com
core@ip-10-74-131-124 sandbox-aquasec-us-east-1a-control ~ $ etcdctl get aqua/config/gateway-external
https://aquasec-sa-AquaElb-PN36MQ2MP0NT-1376431810.us-east-1.elb.amazonaws.com
core@ip-10-74-131-124 sandbox-aquasec-us-east-1a-control ~ $ 
```

**Pull request merged**